lightning/src/ln/channel.rs

```diff
@@ -1534,6 +1534,7 @@ impl<SP: Deref> Channel<SP> where
 				res
 			},
 			_ => {
+				self.phase = phase;
 				debug_assert!(!matches!(self.phase, ChannelPhase::Undefined));
```

Comment:
Ah, the `debug_assert` should have probably gone outside the `match` as is done in `funding_signed`. Guess it doesn't really matter since the other arms should be obviously correct.